checkstyle-tester/diff.groovy

```diff
@@ -823,8 +836,9 @@ class Config {
             checkstyleCfg: baseConfig,
             listOfProjects: listOfProjects,
             destDir: tmpMasterReportsDir,
-            extraMvnRegressionOptions: extraMvnRegressionOptions,
+            extraRegressionOptions: extraRegressionOptions,
             allowExcludes:allowExcludes,
+            continueOnError:continueOnError,
```

Comment:
Fixed a bug in `continueOnError`, where if new option is not listed in these areas, then we cannot reference them in the main execution as we actually reference them back to here.

Without listed them here, doing `cfg.[option]` in main code always returned `null` and it would not fail anywhere since we were treating it as a boolean. It would just never get enabled.